config/xml/xml.go

```diff
@@ -190,7 +190,28 @@ func (c *ConfigContainer) DefaultStrings(key string, defaultval []string) []stri
 // GetSection returns map for the given section
 func (c *ConfigContainer) GetSection(section string) (map[string]string, error) {
 	if v, ok := c.data[section]; ok {
-		return v.(map[string]string), nil
+
+		var interfaceToStr func(map[string]interface{}) map[string]string
+
+		interfaceToStr = func(values map[string]interface{}) map[string]string {
```

Comment:
I agree with @fuxiaohei. set the interfaceToStr as a function. that will be easily to understand. use too much closure. make the code unreadable